mappings/net/minecraft/datafixer/fix/BitStorageAlignFix.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5196 net/minecraft/datafixer/fix/BitStorageAlignFix
```

Comment:
Either `AlignBitStorageFix` or `BitStorageAlignmentFix` would be better. Unless this is a Mojang name?

Reply:
This is indeed a Mojang name.

Reply:
> This is indeed a Mojang name.

Wait, how do you know that 🤔

Reply:
Data fixes contain their names as strings in the classes.